From 5a43a570d3d160d732ba7a509b01e5897162be86 Mon Sep 17 00:00:00 2001
From: Edoardo Marangoni <ecmm@anche.no>
Date: Tue, 21 Jan 2025 09:49:53 +0100
Subject: [PATCH 3/3] Enable exnrefs by default

---
 src/wasm/c-api.cc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/wasm/c-api.cc b/src/wasm/c-api.cc
index 15fceadff2e..a7fc3c1d5e2 100644
--- a/src/wasm/c-api.cc
+++ b/src/wasm/c-api.cc
@@ -455,6 +455,7 @@ WASM_EXPORT auto Engine::make(own<Config>&& config) -> own<Engine> {
                          ? v8::platform::NewSingleThreadedDefaultPlatform()
                          : v8::platform::NewDefaultPlatform(
                                i::v8_flags.wasm_capi_thread_pool_size);
+  i::v8_flags.experimental_wasm_exnref = true;
   v8::V8::InitializePlatform(engine->platform.get());
   v8::V8::Initialize();
 
-- 
2.39.5 (Apple Git-154)

