diff --git a/.gn b/.gn
index afdb6fa1201..8a3f226353c 100644
--- a/.gn
+++ b/.gn
@@ -34,5 +34,5 @@ default_args = {
 # These are the list of GN files that run exec_script. This whitelist exists
 # to force additional review for new uses of exec_script, which is strongly
 # discouraged except for gypi_to_gn calls.
-exec_script_whitelist = build_dotfile_settings.exec_script_whitelist +
+exec_script_whitelist = build_dotfile_settings.exec_script_allowlist +
                         [ "//build_overrides/build.gni" ]
